xblock/test/test_parsing.py

```diff
@@ -19,7 +19,7 @@
 
 def get_namespace_attrs():
     """ Returns string suitable to be used as an xmlns parameters in XBlock XML representation """
-    return " ".join('xmlns:{}="{}"'.format(k, v) for k, v in XML_NAMESPACES.items())
+    return " ".join('xmlns:{}="{}"'.format(k, v) for k, v in sorted(XML_NAMESPACES.items()))
```

Comment:
I like consistently generated XML. Another option, though, would be to compare parsed XML objects.